mermithid/processors/TritiumSpectrum/TritiumSpectrumProcessor.py

```diff
@@ -178,3 +177,75 @@ def definePdf(self, wspace):
         getattr(wspace, 'import')(KE)
         getattr(wspace, 'import')(pdf)
         return wspace
+
+    def _Fit(self):
```

Comment:
Not sure I understand the goal of this function or more precesily what difference there is compared with the one implemented in morpho.
It seems that there are residuals plots also produced here.
This sounds to me like a very interesting feature that you might want to see elsewhere; then it would make sense to implement it inside the morpho RooFitProcessor https://github.com/morphoorg/morpho/blob/master/morpho/processors/sampling/RooFitInterfaceProcessor.py#L142

Reply:
I think you made this processor and I kind of don't want to mess with it :)

Reply:
well then I guess we should trust "past Mathieu" on this: he might have a good reason why he did this...
Maybe this is because this is for one dimensional datasets (only dependence on KE) and therefore it is possible to plot some sort of residuals, while it is not possible for other multidimensional datasets.